### PR TITLE
a11y WCAG 2.1 color contrast updates

### PR DESCRIPTION
**Pull request by npsp-reedestockton**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by npsp-reedestockton** [comment]
[W-9537095](https://gus.lightning.force.com/lightning/r/a07AH000000PxQJYA0/view)
[W-9537133](https://gus.lightning.force.com/lightning/r/a07AH000000PxT8YAK/view)
[W-9537117](https://gus.lightning.force.com/lightning/r/a07AH000000PxS0YAK/view)
[W-14001172](https://gus.lightning.force.com/lightning/r/a07EE00001Z8t04YAB/view)

File: force-app/main/default/pages/MTCH_FindGifts.page [hunk 1]
```diff
@@ -146,9 +146,14 @@
                     <apex:repeat value="{!potentialGifts}" var="g">
                         <tr>
                             <td>
-                                <apex:inputCheckbox value="{!selection[g.Id]}" id="cbGift">
-                                    <apex:actionSupport action="{!refresh}" event="onchange" rerender="totalSelected, totalUn" />
-                                </apex:inputCheckbox>
+                                <div class="slds-form-element__control">
```
Comment (lparrott) [inline force-app/main/default/pages/MTCH_FindGifts.page:149]: Great solution!

File: force-app/main/default/pages/STG_PanelContactRoles.page [hunk 2]
```diff
@@ -52,9 +52,11 @@
...
-                    <apex:inputCheckbox value="{!stgService.stgHH.npo02__Always_Rollup_to_Primary_Contact__c}" rendered="{!isReadOnlyMode}" disabled="true" id="cbxARPCO" html-aria-describedby="{!$Component.cbxARPCHelp}" styleClass="slds-checkbox"/>
-                    <apex:outputPanel id="cbxARPCHelp" layout="block">
+                    <label class="slds-checkbox">
+                        <apex:inputCheckbox value="{!stgService.stgHH.npo02__Always_Rollup_to_Primary_Contact__c}" rendered="{!isEditMode}" id="cbxARPC" html-aria-describedby="{!$Component.cbxARPCHelp}" styleClass="slds-checkbox"/>
+                        <apex:inputCheckbox value="{!stgService.stgHH.npo02__Always_Rollup_to_Primary_Contact__c}" rendered="{!isReadOnlyMode}" disabled="true" id="cbxARPCO" html-aria-describedby="{!$Component.cbxARPCHelp}" styleClass="slds-checkbox"/>
+                    <span class="slds-checkbox_faux"></span>
+                    </label>                                        <apex:outputPanel id="cbxARPCHelp" layout="block">
```
Comment (lparrott) [inline force-app/main/default/pages/STG_PanelContactRoles.page:59]: Missing a new line here? Looks like there's a few of these, I'll let you find/fix the rest 😉

File: force-app/main/default/pages/STG_PanelAllocations.page [hunk 3]
```diff
@@ -64,9 +64,11 @@
...
-                        <apex:inputCheckbox value="{!stgService.stgAllo.Use_Fiscal_Year_for_Rollups__c}" rendered="{!isReadOnlyMode}" disabled="true" id="cbxUFYFRO" html-aria-describedby="{!$Component.cbxUFYFRHelp}" styleClass="slds-checkbox" />
-                        <apex:outputPanel id="cbxUFYFRHelp" layout="block">
+                        <label class="slds-checkbox">
+                            <apex:inputCheckbox value="{!stgService.stgAllo.Use_Fiscal_Year_for_Rollups__c}" rendered="{!isEditMode}" id="cbxUFYFR" html-aria-describedby="{!$Component.cbxUFYFRHelp}" styleClass="slds-checkbox" />
+                            <apex:inputCheckbox value="{!stgService.stgAllo.Use_Fiscal_Year_for_Rollups__c}" rendered="{!isReadOnlyMode}" disabled="true" id="cbxUFYFRO" html-aria-describedby="{!$Component.cbxUFYFRHelp}" styleClass="slds-checkbox" />
+                        <span class="slds-checkbox_faux"></span>
+                        </label>                        <apex:outputPanel id="cbxUFYFRHelp" layout="block">
```
Comment (lparrott) [inline force-app/main/default/pages/STG_PanelAllocations.page:71]: Missing new line?

File: force-app/main/default/pages/STG_PanelTDTM.page [hunk 4]
```diff
@@ -54,7 +54,7 @@
                         <apex:outputLabel value="{!$ObjectType.Trigger_Handler__c.Fields.Class__c.Label}" for="tbxC" styleClass="slds-form-element__label"
                         />
                         <div class="slds-form-element__control">
-                            <apex:inputField value="{!thNew.Class__c}" required="false" id="tbxC" html-aria-describedby="{!$Component.tbxCHelp}" styleClass="fslds-input"
```
Comment (lparrott) [inline force-app/main/default/pages/STG_PanelTDTM.page:57]: I think we both know what the "f" stands for

**Review by lparrott — CHANGES_REQUESTED** [review]
Great work! The bulk of my testing will be spent on the checkbox changes, just because there's a small risk of regression there. I'll approve after testing.

**Review by lparrott — APPROVED** [review]

**Pull request merged**